roles/network_plugin/calico/templates/calico-config.yml.j2

```diff
@@ -56,10 +56,14 @@ data:
             {% else %}
             "ipam": {
               "type": "calico-ipam",
-              {%   if enable_dual_stack_networks %}
+              {% if ipv6_stack and ipv4_stack %}
```

Comment:
could be:

```
if ipv4:
   assign_ipv4: true
else:
  assign_ipv4: false
if ipv6:
     assign_ipv6: true
else:
    assign_ipv6: false
```
or 
```
if ipv4:
  assigne_ipv4=true
fi
if ipv6
  assign_ipv6=true
fi
```

Reply:
the problem is in the comma, in the case of dual_stacke it is needed